spec/javascripts/components/toggle-input-class-on-focus-spec.js

```diff
@@ -23,11 +23,14 @@ describe('A toggle class module', function () {
 
     it('applies the focus style on focus and removes it on blur', function () {
       var searchInput = document.querySelector('.js-class-toggle')
-      expect(searchInput).not.toHaveClass('focus')
+      expect(window.isClassOnElement(searchInput, 'js-class-toggle')).toEqual(true)
```

Comment:
I'm not sure what this line is for? It seems to be testing that the element found on the previous line using the class `js-class-toggle` has that class? Wouldn't the previous line fail if that wasn't the case?

Reply:
@andysellick Would you like me to remove it?

Reply:
Whoops, I didn't realise I introduced it. Will remove it now.